### PR TITLE
54 situation

### PR DESCRIPTION
**Pull request by Sarvesh-Mk**

### CODE REVIEW

File: logic/SituationManager.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import json
 import random
 
+from devtools.debug import debug
```
Comment (yashkir) [inline logic/SituationManager.py]: remove

File: logic/SituationManager.py [hunk 2]
```diff
@@ -10,27 +11,31 @@ class SituationManager():
...
-                situations = json.load(file)
+        elif self.situation_counter >= 5:
 
-            data = situations[self.situation_counter % 5]
+            data = self.situations_cannon[id]
 
         # Get the data for a normal situation
         else:
-            with open('situations/normal.json', 'r') as file:
-                situations = json.load(file)
```
Comment (yashkir) [inline logic/SituationManager.py]: remove this blank line

File: logic/StateManager.py [hunk 3]
```diff
@@ -72,9 +72,12 @@ def apply_stats(self, response):
             self.state[0]['player']['stats'][stat] = self.player_stats.stat_dict[stat]
 
         if self.player_stats.sus_int > 50 or self.player_stats.sad_int > 50:
-            self.game.game_screen.game_over()
+            # self.game.game_screen.game_over()
```
Comment (yashkir) [inline logic/StateManager.py]: why is this commented?

File: situations/landmark.json [hunk 4]
```diff
@@ -27,5 +27,34 @@
             }
         }
     ]
+},
+{
+    "prompt": "the government is coming to take tests on everyone what do you do",
```
Comment (yashkir) [inline situations/landmark.json]: change to " The government is coming to take tests, what do you do?"

**Review by yashkir — CHANGES_REQUESTED** [review]

File: logic/StateManager.py (hunk 3 above)
Comment (Sarvesh-Mk) [reply logic/StateManager.py]: its cause im bad at the game and i was trying to debug something lol

File: situations/landmark.json (hunk 4 above)
Comment (Sarvesh-Mk) [reply situations/landmark.json]: alright

**Review by yashkir — APPROVED** [review]

**Pull request merged**